### PR TITLE
Removed way to generate emails by creating_email.py

### PR DESCRIPTION
**Pull request by alexbeatnik**
We had two ways to generate emails by module memesis and by
class CreatingFullEmail this commit leave only one way
creating emails by memesis. In all files where used class
CreatingFullEmail way was changing to memesis.
Deleted file oct/tests/web/creating_emails.py
and tests/test_emailsGeneration.py

### CODE REVIEW

File: oct/pages/account.py [hunk 1]
```diff
@@ -7,8 +7,8 @@ class AccountPage(Page):
     def __init__(self, browser: Remote) -> None:
         self._browser = browser
 
-    def open(self, device: Device) -> None:
```
Comment (extsoft) [inline oct/pages/account.py]: @alexbeatnik why did you rename this method?

File: oct/pages/account.py [hunk 2]
```diff
@@ -7,8 +7,8 @@ class AccountPage(Page):
...
+    def open_page(self, device: Device) -> None:
         self._browser.get(f"https://{device.connections.main.ip}/index.php?route=account/account")
 
-    def loaded(self) -> bool:
+    def loaded_page(self) -> bool:
```
Comment (extsoft) [inline oct/pages/account.py]: @alexbeatnik why did you rename this method?

File: code-assessment.sh [hunk 3]
```diff
@@ -10,7 +10,6 @@ black --check . || add_fail black
 pylint oct suite.py || add_fail pylint
 flake8 oct suite.py || add_fail flake8
 mypy oct suite.py || add_fail mypy
-py.test -v tests || add_fail py.test
```
Comment (extsoft) [inline code-assessment.sh]: @alexbeatnik can we create a unit test(s) for some other code in order to keep unit testing on?

File: oct/pages/account.py (hunk 1 above)
Comment (alexbeatnik) [reply oct/pages/account.py]: I had an error in flake8 "python builtin, consider renaming the class attribute" code in flake8 A003

File: oct/pages/account.py (hunk 2 above)
Comment (alexbeatnik) [reply oct/pages/account.py]: @extsoft  I had an error in flake8 "python builtin, consider renaming the class attribute" code in flake8 A003. This renaming fixed it

File: code-assessment.sh (hunk 3 above)
Comment (alexbeatnik) [reply code-assessment.sh]: Fixed created test for module mimesis

File: oct/pages/account.py (hunk 1 above)
Comment (alexbeatnik) [reply oct/pages/account.py]: Fixed

Comment (alexbeatnik) [reply oct/pages/account.py]: @extsoft Fixed

File: oct/pages/account.py (hunk 2 above)
Comment (alexbeatnik) [reply oct/pages/account.py]: @extsoft Fixed

File: .flake8 [hunk 4]
```diff
@@ -39,3 +39,5 @@ ignore =
     # As we use "W504 line break after binary operator", this has to be disabled. More on
     # https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
     W503,
+    # python builtin, consider renaming the class attribute
```
Comment (extsoft) [inline .flake8]: @alexbeatnik why did you disable this rule?

Comment (alexbeatnik) [reply .flake8]: @extsoft  I had an error in flake8 "python builtin, consider renaming the class attribute" code in flake8 A003. This can be fixing by renaming classes 'open' and 'loaded' or by disabling this rule. I think that you dont like version with renaming so i disable this rule

Comment (alexbeatnik) [reply .flake8]: Fixed, by changing version of pydocstyle

Comment (extsoft) [reply .flake8]: @extsoft could you please explain why this error appears only now? We are using the current setup for a while.

Comment (alexbeatnik) [reply .flake8]: @extsoft pydocstyle was updated 18 days ago you can see it on the page of it project, (https://github.com/PyCQA/pydocstyle/) and we have no version of the pydocstyle in  	requirements-dev.txt so pip installed last version. Last commit in our project was more than 40 days ago, version of module changed so we have this error

Comment (extsoft) [reply .flake8]: @alexbeatnik thank you for the explanation. So, please create another PR which will rename `open` to `load`, `loaded` to `available`. After it will be accepted, please add these changes to this branch. And we will review it again.

Comment (alexbeatnik) [reply .flake8]: @extsoft Thanks for review, i created new PR
